cpu/sam0_common/periph/spi.c

```diff
@@ -117,6 +118,10 @@ int spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
     /* power on the device */
     poweron(bus);
 
+    /* disable the device */
+    dev(bus)->CTRLA.reg &= ~(SERCOM_SPI_CTRLA_ENABLE);
+    while (dev(bus)->SYNCBUSY.reg & SERCOM_SPI_SYNCBUSY_ENABLE) {}
```

Comment:
If I am understanding you right, a counterexample is that if I put the disable inside the spi_release, but leave the disable also in spi_acquire, the power consumption goes up again. So I don't think it is the fact that spi_acquire is being called while the device is enabled.